packages/lesswrong/lib/collections/posts/fragments.ts

```diff
@@ -54,6 +54,18 @@ registerFragment(`
     commentsLocked
     commentsLockedToAccountsCreatedAfter
 
+    # does this go here??? *voiceless screaming*
```

Comment:
I think it could happily go on PostsDetails? You only use it on PostsPage, which uses (through some inheritance) PostsDetails.

Reply:
If not, I would consider this more a comment for a PR than for the code. (Nit)

Reply:
Yeah, that was mostly a reminder to myself to leave a PR comment later (which I then forgot to do, but at least someone noticed!)